iOS/third-party-podspecs/ComponentKit.podspec

```diff
@@ -17,5 +17,5 @@ Pod::Spec.new do |s|
     'CLANG_CXX_LANGUAGE_STANDARD' => 'gnu++14',
     'CLANG_CXX_LIBRARY' => 'libc++',
   }
-  s.dependency 'Yoga'
+  s.dependency 'Yoga', '~> 1.6'
```

Comment:
This should be `1.8`. https://cocoapods.org/pods/Yoga

Reply:
I added 1.6 since ComponentKit podspec in componentkit repo points to 1.6.0. I will change it to 1.8 since it should have no impact